README.md

```diff
@@ -148,13 +148,18 @@ public showToast() {
 }
 ```
 
-Finally, you can show a flyout using the `showFlyout` method:
+Finally, you can show a flyout using the `showFlyout` method and close a flyout using the `closeFlyout` method:
 
 ```js
 this.addinClientService.showFlyout({
   url: someUrl,
   context: someContextObject
-});
+}).subscribe(() => {
+  // Define what happens when a flyout has closed
+});;
```

Comment:
extra ;

also - I might remove the closeFlyout() example code below (and the "...and close a flyout...") sentence above, just to avoid making it sound like you *have* to use it.  I.e., you don't need to manually close the flyout normally since it will happen automatically.

Reply:
Agreed.  Thanks!